config/testgrids/kubernetes/sig-node/config.yaml

```diff
@@ -53,6 +53,10 @@ dashboards:
 
 - name: sig-node-cri-o
   dashboard_tab:
+    - name: crio-NodeConformance-cgroupv1
```

Comment:
@dims I am not sure if we need this here if we are specifying `testgrid-tab-name` above in the test jobs.

Reply:
I will remove it, if we don't need it.